propagation/b3/http.go

```diff
@@ -21,52 +21,104 @@ import (
 	"github.com/openzipkin/zipkin-go/propagation"
 )
 
+type InjectOption func(opts *InjectOptions)
+
+type InjectOptions struct {
+	shouldInjectSingle   bool
+	shouldInjectMultiple bool
+}
+
+// WithInjectSingleAndMultiple allows to include both single and multiple
+// headers in the context injection
+func WithInjectSingleAndMultiple() InjectOption {
```

Comment:
Multiple is ambiguous here... as it is now I find it unclear if it means multiple headers (both single B3 and multi B3) or simply multi-header B3...

I'd rename to `WithSingleAndMultiHeader`